lib/bloc/invoice/invoice_bloc.dart

```diff
@@ -36,6 +33,7 @@ class InvoiceBloc {
   Stream<bool> get paidInvoicesStream => _paidInvoicesController.stream;
 
   Int64 payBlankAmount = Int64(-1);
+  String description = "";
```

Comment:
What is the purpose of this "description" field? I also don't see it assigned anywhere.